### PR TITLE
Make converter executable from node & handle object types

### PR DESCRIPTION
**Pull request by AugustinLF**
Hi, good job for this library! While trying to integrate it with one of my servers, I've stumbled onto two problems:
- It's not possible to call this library from node, which is something I'd like to do
- The type conversion works only one level deep, it you schema has properties that requires object, the converter errors. 

This PR solves these two problems. If you'd like me to split it, it's doable (not fun, but I can do it if it helps^^).

I tried to follow your implementation as much as possible (hence the recursive calls to `fieldsFromSchema`). If you'd like me to change that, feel free to do so.

In the process I added a more `e2e` test in `test/jsonSchemasToGraphqlSchema.js`, since I was debugging from there, I can remove it (or test `src/index` than the converter if you'd prefer).

Sorry for the yarn.lock, I'm on `yarn@1.10`. I can reset it if you'd like.

### TODO
- [ ] The codecoverage is not at 100%, I'll fix that if I'm sure you're interested in the PR^^
- [ ] arrays of objects don't work either, but it should probably done in another PR.

### CODE REVIEW

File: src/converter.js [hunk 1]
```diff
@@ -105,7 +105,16 @@ function fieldsFromSchema (context, parentTypeName, schema, buildingInputType) {
 
   const fields = mapValues(schema.properties, function (attributeDefinition, attributeName) {
     const qualifiedAttributeName = `${parentTypeName}.${attributeName}`;
-    const type = mapType(context, attributeDefinition, qualifiedAttributeName, buildingInputType);
+    let type;
+    if (attributeDefinition.type === 'object') {
```
[the remark this answers is not in the document]
Comment (AugustinLF) [reply src/converter.js]: Would it be fine for you if I also push the support for arrays of objects, and definitions (see the [array of things example](https://json-schema.org/learn/miscellaneous-examples.html) in this PR?^^ I continued working on the project, and had to update the code related to the recursion, so some things might have to change again!

I mean, I make the requested changes, but I'll have to immediately change the code again, and if I could avoid that, that'd be nice.

[the remark this answers is not in the document]
Comment (AugustinLF) [reply src/converter.js]: Sure, I'll do so!

**Comment by AugustinLF** [comment]
I pushed again. I deleted the extra test file, since right now, it doesn't add any value over the existing test plan. I might reintroduce it later, but I had problems to test some error cases.

**Comment by mdlavin** [comment]
I published this as v1.1.0 into npm

**Pull request merged**